Automattic/jetpack
# General: update Jetpack brand color to new reference

## Proposed changes:

This old color code isn't used anymore; let's switch to the new color instead.

### Other information:


## Jetpack product discussion

* No

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Not much to test here; check that the color code matches what we use elsewhere in the plugin
